worker/backend/integration/run

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+readonly CONTAINER_NAME="containerd-integration"
+readonly CONCOURSE_DIR="$(realpath $(dirname $0)/../../..)"
```

Comment:
@pivotal-jamie-klassen thanks for the tip! btw, there's still the need for `realpath` because `--volume` needs an absolute path: https://github.com/docker/cli/issues/1203